planner/python_poetry_planner.go

```diff
@@ -136,3 +124,17 @@ func getPythonImage(version *version) string {
 	}
 	return fmt.Sprintf("python:%s-slim", version.exact())
 }
+
+func (g *PythonPoetryPlanner) isBuildable(srcDir string) (bool, string) {
+	project := g.PyProject(srcDir)
+	if project == nil {
+		return false, "Project is not buildable: pyproject.toml not found"
```

Comment:
"Project is not buildable" feels to me like the interface of the planner is leaking into the messaging we give the user.

What about:
```
Could not build container for python application.
  pyproject.toml is missing and needed to install python dependencies.
```